include/miopen/config.h.in

```diff
@@ -78,11 +78,44 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
+#cmakedefine01 MIOPEN_OFFLINE_COMPILER_PATHS_V2
+#if MIOPEN_OFFLINE_COMPILER_PATHS_V2
+// Include rocm-core header for get ROCm Install Path Method
+#include <rocm-core/rocm_getpath.h>
```

Comment:
This file should not include any other header file.

Reply:
Thanks for comments, Updated this in latest version from https://github.com/arvindcheru/MIOpen/pull/5